Automattic/wp-calypso
# Purchases: Update the list view design

See #12746 — This PR implements the new purchases list design. There should be no functionality change in this PR, though some bigger changes were made for passing lint (added localize, and renamed folders to pass the CSS classname check).

**Current**

![screen shot 2017-04-05 at 1 48 01 pm](https://cloud.githubusercontent.com/assets/541093/24719137/a5cce814-1a06-11e7-8d96-6dde6ac7a2fb.png)

**Updated styles**

![screen shot 2017-04-05 at 3 11 27 pm](https://cloud.githubusercontent.com/assets/541093/24722602/5a581c8a-1a12-11e7-88e1-eeec6727c9fe.png)

Domain mapping/domain products use the domains gridicon…

![screen shot 2017-04-05 at 3 14 31 pm](https://cloud.githubusercontent.com/assets/541093/24722671/99a11c16-1a12-11e7-9246-62ee70f4dcb0.png)

…while legacy products don't show an icon.

**To test**

1. Visit [/me/purchases](https://calypso.live/me/purchases?branch=update/purchases-list-design-i2)
2. Make sure it looks good 😄
